bower.json

```diff
@@ -24,6 +24,8 @@
     "angular-ui-validate": "1.1.1",
     "bootstrap": "3.3.4",
     "json-refs": "~0.2.0",
+    "jquery": "latest",
```

Comment:
We should use fixed versions, otherwise users may be prompted to select the correct version of a framework as it was the case for me.